dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -646,6 +646,70 @@ open class Jpql : JpqlDsl {
         return Expressions.ln(value.toExpression())
     }
 
+    /**
+     * Creates an expression that represents the mod of values.
+     */
+    @SinceJdsl("3.4.0")
+    fun <T : Any> mod(expr: KProperty1<T, Int>, value: Int): Expression<Int> {
```

Comment:
It appears that the mod has not been implemented, but the commit appears to have been distorted.

Reply:
<img width="341" alt="image" src="https://github.com/line/kotlin-jdsl/assets/66164361/727771e5-60ba-439e-8e8d-0f74800aad8f">

Oops, I had them in the order they appear in the official documentation, which seems to have caused some confusion. I've now fixed them back to their original position.